index.html

```diff
@@ -215,5 +215,72 @@ <h3>Skills</h3>
         </ul>
       </section>
     </main>
+    <footer>
+      <form action="https://formspree.io/f/xayvlegd" method="post">
+        <ul class="form-container">
+          <li class="contactme-head">
+            <h3>
+              I'm always interested in hearing about new projects, so if you'd
+              like to chat please get in touch.
+            </h3>
+          </li>
+          <li>
+            <input
+              type="text"
+              name="name"
+              id="user_name"
+              placeholder="Full name"
+              maxlength="30"
+              required
+            />
+          </li>
+          <li>
+            <input
+              type="email"
+              name="email"
+              id="user_email"
+              placeholder="Email address"
+              required
+            />
+          </li>
+          <li>
+            <textarea
+              name="message"
+              id="user_text"
+              cols="30"
+              rows="10"
+              maxlength="500"
+              placeholder="Enter text here"
+              required
+            ></textarea>
+          </li>
+          <li class="submit-button-container">
+            <button class="submit-contact" type="submit">Get in touch</button>
+          </li>
+        </ul>
+      </form>
+      <div>
+        <ul class="footer-icons">
+          <li>
+            <a href="https://twitter.com/NatiDemelash"
+              ><img class="icons" src="images/twitter.png" alt="twitter"
+            /></a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/natnael-demelash/"
+              ><img class="icons" src="images/Linkedin.png" alt="Linkedin"
+            /></a>
+          </li>
+          <li>
+            <a href="https://github.com/NatiDeme"
+              ><img class="icons" src="images/github.png" alt="github"
+            /></a>
+          </li>
+        </ul>
```

Comment:
[OPTIONAL] Please kindly add all social media icons just like in the figma template you picked. This is needed for you will be using this as your main portfolio so please dont over look things even the smallest details are important.